applets/notification_area/main.c

```diff
@@ -186,6 +186,16 @@ static const GtkActionEntry menu_actions [] = {
 	  G_CALLBACK (about_cb) }
 };
 
+static void
+na_tray_applet_size_allocate (GtkWidget     *grid,
+                              GtkAllocation *allocation,
+                              gpointer       data)
+{
+  NaTrayApplet *applet = data;
+  g_return_if_fail (NA_IS_GRID (grid));
+
+  set_grid_display_mode (NA_GRID (grid), MIN_ICON_SIZE_DEFAULT);
```

Comment:
Hmm, why this in size-allocate if min size does not depend on allocation? I think you can probably call this after creating new grid or even do this in `na_grid_init`, seems that you already do that.

I guess you want this here to call `refresh_grid`, right? But then you get size-allocate signal on grid itself so you should be able to handle that in `na-grid.c`.